index.html

```diff
@@ -523,6 +523,10 @@ <h3 id="miscellaneous"> Miscellaneous </h3>
     <td><img src='https://img.shields.io/codeclimate/coverage/github/triAGENS/ashikawa-core.svg' alt=''/></td>
     <td><code>https://img.shields.io/codeclimate/coverage/github/triAGENS/ashikawa-core.svg</code></td>
   </tr>
+  <tr><th> Code Climate: </th>
+    <td><img src='https://img.shields.io/codeclimate/issues/github/me-and/mdf.svg' alt=''/></td>
+    <td><code>https://img.shields.io/codeclimate/issues/github/me-and/mdf.svg</code></td>
+  </tr>
```

Comment:
index.html is automatically generated. Could you move that modification to try.html instead, please?